src/Components/MainLayout/MainLayout.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react'
 import Navigation from './Navigation'
 import Container from './Container'
+import AuthProvider from '../../Providers/AuthProvider'
 
 const MainLayout = () => {
     return (
-        <>
-            <Navigation/>
-            <Container/>
-        </>
+        <AuthProvider>
+            <Navigation />
+            <Container />
+        </AuthProvider>
```

Comment:
Navigation and Container components would be handled in the Router (as the example on the App component) so for the MainLayout component to render its children you could do something like this:

```javascript
const MainLayout = ({ children }) => {
  return <AuthProvider>{children}</AuthProvider>;
};
```